server/api.js

```diff
@@ -5,6 +5,8 @@ import axios from "axios";
 import jobsData from "../data.json";
 import myRouteHandler from "./utils/refreshData";
 const router = Router();
+import arbeitNowRouteRH from "./utils/Sana/arbeitNowRoute";
```

Comment:
@SanaAsaf good start, update line to reflect the file path and its good to merge

import arbeitNowRouteRH from "./utils/arbeitNowRoute"